### PR TITLE
Feature signup currency

### PR DESCRIPTION
**Pull request by JamieDraper**
Feature for the donate ask slide on scrolling signup pages. Updated to show the users currency and converted donate button amounts, based on the country they select in the signup form.

**afteraction-share.html**
- we make an api request to https://api.exchangeratesapi.io (if this fails, we default to hardcoded exchange rates that aren't up to date)
- we covert the donate amount and change the currency for users in Canada, Australia, New Zealand, United Kingdom and Europe. Otherwise we default to USD
- we pass in a currency URL parameter to the donate page (used to preselect currency)

**donate.html**
- if we have the currency parameter in the URL, we use it to preselect a currency on the donate page

### CODE REVIEW

File: afteraction-share.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Custom template for same-page post-AJAX submit share ask.
 {% endcomment %}
 
+{{page}}
```
Comment (mrkimani-ndegwa) [inline afteraction-share.html:5]: Do we need this guy here or am I missing something?

File: afteraction-share.html [hunk 2]
```diff
@@ -185,6 +193,60 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
...
+				'Brazil': {
+					currencySymbol: 'R$',
+					currency: 'BRL',
+					exchangeRate: exchangeRateData['BRL'] || 5.07
+				}
+			};
+			if (currencyData[userLocation]) {
+				$('#afteraction-donate-inner .afteraction-donate-amount-container a').each(function(i){
+					// replace amount and add currency param in donate button urls
+					var $this = $(this);
```
Comment (mrkimani-ndegwa) [inline afteraction-share.html:238]: I am a sucker for const over var. Could we consider the same here?

Comment (JamieDraper) [reply afteraction-share.html:237]: Good call, thanks Kimani! I've amended in my commit

File: afteraction-share.html (hunk 1 above)
Comment (JamieDraper) [reply afteraction-share.html]: I believe this was in the template I originally cloned when working on this back in March! I've tried removing it from my test page and can't see any problems. I've removed it and made another commit to this branch

**Pull request merged**